### ISSUE TITLE
"manual imagery" descriptions not translatable

### ISSUE THREAD

**Issue by manfredbrandl** [openstreetmap/iD#7724]
![image](https://user-images.githubusercontent.com/5360273/85298776-c91d5380-b4a4-11ea-9e9c-6242f562ce31.png)
![image](https://user-images.githubusercontent.com/5360273/85298902-eb16d600-b4a4-11ea-8968-cc79a52d3f62.png)

**Comment by maro-21** [comment]
I reported it four months ago in https://github.com/openstreetmap/iD/issues/7358 and it was closed as solved.

**Comment by manfredbrandl** [comment]
The strings All fields, All tags and All relations are translatable now, so #7358 is solved, but not for the maxar strings. I hope "Somebody" will help here.

**Comment by quincylvania** [comment]
Turns out the Maxar strings have a different underlying issue from the other strings in #7358. Since they use encrypted endpoints, they're not included in the editor-layer-index and so don't use the standard imagery translation workflow.

**Comment by manfredbrandl** [comment]
@quincylvania no maxar strings in transifex as you have probably seen and reopened

**Comment by quincylvania** [comment]
> no maxar strings in transifex as you have probably seen and reopened

@manfredbrandl Yep. I mistakenly thought Transifex pulled the strings from `en.json`, but it actually pulls the imagery and community strings right from their respective projects. This is okay, but it means we'll probably have to include the Maxar strings in `core` instead of `imagery`.

<img width="670" alt="Screen Shot 2020-06-24 at 11 50 18 AM" src="https://user-images.githubusercontent.com/2046746/85596090-7021fc80-b617-11ea-95bf-b766e9ef6087.png">

**Comment by manfredbrandl** [comment]
Including maxar string in core not imagery: not the best but the best available solution so far. very okay for me.

**Comment by maro-21** [comment]
- There should be a space between _50_ and _cm_
- Why are there two Maxar layers if they have the same imagery?
- Why are there two Esri layers if they have the same imagery?

**Comment by quincylvania** [comment]
> Why are there two Maxar layers if they have the same imagery?
> Why are there two Esri layers if they have the same imagery?

@maro-21 They're not the same imagery all over the globe. Both companies maintain two separate endpoints and can independently update the data for any given location at any time.

**Comment by tyrasd** [comment]
Currently, the Maxar imagery is unavailable for mapping (see #9710), but the issue applies in principle to all sources in `data/manual_imagery.json`. ~~Currently, these are the two Mapbox layers: _Mapbox Satellite_ and the _Locator Overview_ overlay)~~ Affected are only sources which are not also present in [ELI](https://github.com/osmlab/editor-layer-index/)[^1], so currently, no layers are affected by this bug, so a workaround to avoid this issue would be to always add iD's manual imagery also to ELI (excluding any potentially iD specific details like API keys, etc.).

[^1]: The iD project on Transifex is configured to pull the to-be-translated strings for the "imagery" resource directly from the ELI repository. Because of that, iD's build scripts cannot simply "add" the manual imagery to the translation source files.
